client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -263,6 +268,18 @@ class Upload extends React.Component {
 		);
 	}
 
+	renderNotAvailable() {
+		return (
+			<EmptyContent
+				title={ this.props.translate( 'Sorry, themes upload is not yet available for multi-site installations.' ) }
+				line={ this.props.translate( 'Open your site:' ) }
+				action={ this.props.translate( '[WP Admin]' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **44** times:
`translate( 'WP Admin' )` *ES Score: 13.62*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/disable-upload-for-multisite)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).